apps/cdd-backend/src/cdd-worker/cdd.processor.ts

```diff
@@ -109,50 +153,75 @@ export class CddProcessor {
       state,
     });
 
-    if (state === 'restarted') {
-      this.logger.debug('handling netki restart', { jobId });
-      if (individualAddress) {
-        await this.handleNetkiRestart(jobId, individualAddress, netki);
-      } else if (businessApplication) {
-        await this.handleNetkiRestartForBusiness(
-          jobId,
-          businessApplication,
-          netki
-        );
-      }
-    } else if (state === 'completed') {
-      this.logger.debug('handling netki completed', { jobId });
-      if (individualAddress) {
-        await this.createCddClaim(jobId, individualAddress, 'netki');
-        await this.clearAddressApplications(jobId, individualAddress);
-      } else if (businessApplication?.address) {
-        if (!business) {
-          this.logger.error(
-            'no business ID was in callback for business application',
-            { jobId, address: businessApplication.address }
+    switch (state) {
+      case 'restarted':
+        this.logger.debug('handling netki restart', { jobId });
+        if (individualAddress) {
+          await this.handleNetkiRestart(jobId, individualAddress, netki);
+        } else if (businessApplication) {
+          await this.handleNetkiRestartForBusiness(
+            jobId,
+            businessApplication,
+            netki
           );
+        }
+        break;
+
+      case 'completed':
+        this.logger.debug('handling netki completed', { jobId });
+        if (individualAddress) {
+          await this.createCddClaim(jobId, individualAddress, 'netki');
+          await this.clearAddressApplications(jobId, individualAddress);
+        } else if (businessApplication?.address) {
+          if (!business) {
+            this.logger.error(
+              'no business ID was in callback for business application',
+              { jobId, address: businessApplication.address }
+            );
+
+            throw new Error(
+              'No business ID was in callback, but it was expected'
+            );
+          }
 
-          throw new Error(
-            'No business ID was in callback, but it was expected'
+          await this.redis.setBusinessIdToAddress(
+            business,
+            businessApplication.address
+          );
+        } else {
+          this.logger.info(
+            'no address was associated to netki business application, no business ID association will be made'
           );
         }
-
-        await this.redis.setBusinessIdToAddress(
-          business,
-          businessApplication.address
-        );
-      } else {
+        break;
+
+      case 'hold':
+        await this.slackApp.client.chat.postMessage({
+          token: process.env.SLACK_BOT_TOKEN,
```

Comment:
```suggestion
          token: this.slackBotToken
```

Also consider making a private function to dedupe the boiler plate and accepts `title: string, body: string`